### ISSUE TITLE
Doesn't bundle files

### ISSUE THREAD

**Issue by AngeloCore** [huozhi/bunchee#254]
index.js
```js
const wsh = require("./bruh.js");
const sleep = (ms) => new Promise((r) => setTimeout(r, ms));

fush();

async function fush() {
  while (true) {
    console.log(new Date(), wsh);
    await sleep(500);
  }
}
```

bruh.js
```js
module.exports = "yee bruh";
```

Then i run:
```sh
npx bunchee ./index.js -o ./dist/index.js --no-external -f cjs --runtine node
```

<details>

<summary>dist/index.js</summary>

```js
function asyncGeneratorStep(gen, resolve, reject, _next, _throw, key, arg) {
    try {
        var info = gen[key](arg);
        var value = info.value;
    } catch (error) {
        reject(error);
        return;
    }
    if (info.done) {
        resolve(value);
    } else {
        Promise.resolve(value).then(_next, _throw);
    }
}
function _async_to_generator(fn) {
    return function() {
        var self = this, args = arguments;
        return new Promise(function(resolve, reject) {
            var gen = fn.apply(self, args);
            function _next(value) {
                asyncGeneratorStep(gen, resolve, reject, _next, _throw, "next", value);
            }
            function _throw(err) {
                asyncGeneratorStep(gen, resolve, reject, _next, _throw, "throw", err);
            }
            _next(undefined);
        });
    };
}
const wsh = require("./bruh.js");
const sleep = (ms)=>new Promise((r)=>setTimeout(r, ms));
fush();
function fush() {
    return _fush.apply(this, arguments);
}
function _fush() {
    _fush = _async_to_generator(function*() {
        while(true){
            console.log(new Date(), wsh);
            yield sleep(500);
        }
    });
    return _fush.apply(this, arguments);
}
```

</details>

As you can see, `bruh.js` is __not__ included in the dist file, am I using bunchee wrong?

**Comment by huozhi** [comment]
Taking a look now

**Comment by huozhi** [comment]
Turns out the cjs is not bundled correctly, fixed in v3.8.0. Please upgrade

And `--no-external` arg is not needed, and `--runtine` arg should be `--runtime`.
